pom.xml

```diff
@@ -160,8 +160,8 @@
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-compiler-plugin</artifactId>
         <configuration>
-          <source>1.7</source>
-          <target>1.7</target>
+          <source>11</source>
```

Comment:
Hi @jonnio. I'm a little bit concerned in regards to changing this from 1.7 to 11. I presume that this change will cause newly compiled release to be compatible with Java 11+?

Reply:
Can we stick with Java 8 as long as we do not have any dependencies to higher java versions?